README.markdown

```diff
@@ -2,33 +2,22 @@
 
 Learn the faces of your fellow Recursers!
 
-## Local set up
```

Comment:
It looks like this is accidentally reverting the README changes I made from pairing with you on getting the project set up on your computer.

Reply:
Sorry about that! Not sure how I keep pulling in or reverting old changes during my rebase, but it looks like I resolved it now with the latest commit.